src/getCoinCombination.test.js

```diff
@@ -1,10 +1,41 @@
 'use strict';
 
-describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
+describe('getHumanAge', () => {
+  const { getHumanAge } = require('./getHumanAge');
```

Comment:
The `getHumanAge` function is being imported and tested, but the task requires testing the `getCoinCombination` function. Ensure you import the correct function.